pages/star-search/index.tsx

```diff
@@ -4,7 +4,7 @@ import { MdOutlineSubdirectoryArrowRight } from "react-icons/md";
 import { useEffect, useRef, useState } from "react";
 
 import Image from "next/image";
-import Markdown from "react-markdown";
+import ReactMarkdown from "react-markdown";
```

Comment:
The name doesn't matter as it's a default export. I put it back to what it was with your other changes and it still behaves the same/better with the new changes to formatting that you made elsewhere.

Reply:
Makes sense, makes sense - thanks!!